lib/scripts/configAuditor.js

```diff
@@ -47,78 +44,555 @@ function configAudit (appDir) {
-    Object.keys(userScripts).forEach((userScript) => {
-      if (startScripts.production.includes(userScript) === true) {
-        hasProdScript = true
-      }
-      if (startScripts.development.includes(userScript) === true) {
-        hasDevScript = true
-      }
-    })
-    if (userScripts[defaultScript] === undefined) {
-      if (defaultScripts[defaultScript].priority === 'error') {
-        logger.error('⚠️', ` Missing script "${defaultScript}"!`.bold)
-        errors = true
-      } else if (defaultScripts[defaultScript].priority === 'warning') {
-        logger.warn(`Missing recommended script "${defaultScript}"!`.bold)
-        errors = true
-      }
-    } else if (defaultScripts[defaultScript].inspect === true) {
-      if (userScripts[defaultScript].includes('roosevelt') && userScripts[defaultScript] !== defaultScripts[defaultScript].value) {
-        logger.error('⚠️', ` Detected outdated script "${defaultScript}". Update contents to "${defaultScripts[defaultScript].value}" to restore functionality.`.bold)
-        errors = true
-      }
+
+  defaultScriptKeys.forEach((script) => {
+    switch (script) {
+      case 'start':
+      case 'production':
+      case 'prod':
+      case 'p':
+        if (userScripts[script] !== undefined) {
+          hasProdScript = true
+        }
+        break
+      case 'development':
+      case 'dev':
+      case 'd':
+        if (userScripts[script] !== undefined) {
+          hasDevScript = true
+        }
+        break
+      case 'dev-install':
+        checkMissingScript(userScripts, script)
+        checkOutdatedScript(userScripts, script)
+        break
+      case 'di':
+        break
+      case 'dev-prune':
+        checkMissingScript(userScripts, script)
+        checkOutdatedScript(userScripts, script)
+        break
+      case 'dp':
+        break
+      case 'kill-validator':
+        checkMissingScript(userScripts, script)
+        checkOutdatedScript(userScripts, script)
+        break
+      case 'kv':
+        break
+      case 'clean':
+        checkMissingScript(userScripts, script)
+        checkOutdatedScript(userScripts, script)
+        break
+      case 'c':
+        break
+      case 'config-audit':
+        checkMissingScript(userScripts, script)
+        checkOutdatedScript(userScripts, script)
+        break
+      case 'a':
+        break
+      case 'standard':
+        break
+      case 'stylelint':
+        break
+      case 'lint':
+      case 'l':
+        break
+      case 'precommit':
+      case 'pc':
+        break
+      case 'test':
+        if (userScripts[script] === undefined) {
+          logger.warn(`Missing recommended script "${script}"!`.bold)
+          errors = true
+        }
+        break
     }
   })
 
   // output warning for missing start script
   if (!hasProdScript) {
-    logger.warn('Missing production run script(s).'.bold)
+    logger.warn(' Missing production run script(s).'.bold)
```

Comment:
Extra space here on accident or to properly align the content with the emojis on the left? Is this properly formatted when emojis are disabled?

Reply:
The space was to align the text with other `logger.warn` and `logger.error` statements. All of them have a space before the text